specification/mediaservices/resource-manager/Microsoft.Media/stable/2022-08-01/AssetsAndAssetFilters.json

```diff
@@ -468,7 +478,41 @@
           "$ref": "#/definitions/TrackBase"
         }
       ],
-      "properties": {},
+      "properties": {
+        "fileName": {
+          "type": "string",
+          "description": "The file name to the source file. This file is located in the storage container of the asset."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the audio track on a video player. In HLS, this maps to the NAME attribute of EXT-X-MEDIA."
+        },
+        "languageCode": {
+          "type": "string",
+          "description": "The RFC5646 language code for the audio track."
+        },
+        "hlsSettings": {
+          "$ref": "#/definitions/HlsSettings",
+          "description": "The HLS specific setting for the audio track."
+        },
+        "dashSettings": {
```

Comment:
This the dashSettings and below properties only available for Audio track. Text tracks doesn't need this dashSettings?

Reply:
Yes, this is by design. We do not plan to add dash for text track this time.